lib/core/src/Cardano/Wallet/DB/Store/Wallets/Model.hs

```diff
@@ -90,3 +126,79 @@ linkedTransactions (TxMetaHistory m) = Map.keysSet m
 
 walletsLinkedTransactions :: Map W.WalletId TxMetaHistory -> Set TxId
 walletsLinkedTransactions = Set.unions . toList .  fmap linkedTransactions
+
+mkTransactionInfo :: Monad m
+    => TimeInterpreter m
+    -> W.BlockHeader
+    -> TxRelationF 'With
+    -> DB.TxMeta
+    -> m WT.TransactionInfo
+mkTransactionInfo ti tip TxRelationF {..} DB.TxMeta {..} = do
+    txTime <- interpretQuery ti . slotToUTCTime $ txMetaSlot
+    return
+        $ WT.TransactionInfo
+        { WT.txInfoId = getTxId txMetaTxId
+        , WT.txInfoFee = WC.Coin . fromIntegral <$> txMetaFee
+        , WT.txInfoInputs = mkTxIn <$> ins
+        , WT.txInfoCollateralInputs = map mkTxCollateral collateralIns
```

Comment:
```suggestion
        , WT.txInfoCollateralInputs = mkTxCollateral <$> collateralIns
```

Style: We started using it in `txInfoInputs`, so why not here as well. 🤷

Reply:
The rule was: do not touch what you do not need to change. But we will make some exceptions for the sake of evolution!

Reply:
Ah. Fair enough. Though the `filter` was whisked away, so I think it counts as "touched". 🤓